spring-kafka-2/src/main/java/com/baeldung/spring/kafka/multiplelisteners/BookConsumer.java

```diff
@@ -0,0 +1,26 @@
+package com.baeldung.spring.kafka.multiplelisteners;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BookConsumer {
+    private static final Logger LOGGER = LoggerFactory.getLogger(BookConsumer.class);
```

Comment:
Although is static final in this case logger is not a constant so please use lowercase variable for non-constant variable.